src/service/viewport-impl.js

```diff
@@ -134,6 +134,10 @@ export class Viewport {
 
     this.binding_.onScroll(this.scroll_.bind(this));
     this.binding_.onResize(this.resize_.bind(this));
+
+    this.onScroll(() => {
+      this.viewer_.postScroll(this.getScrollTop());
```

Comment:
@dvoytenko What do you mean by once per animation frame? Something like this? https://developer.mozilla.org/en-US/docs/Web/Events/scroll